worker/src/main/scala/org/apache/celeborn/service/deploy/worker/PushDataHandler.scala

```diff
@@ -726,17 +735,14 @@ class PushDataHandler(val workerSource: WorkerSource) extends BaseMessageHandler
       client: TransportClient,
       message: RequestMessage,
       requestId: Long,
-      handler: () => Unit): Unit = {
+      handler: () => Unit,
+      callback: RpcResponseCallback): Unit = {
     try {
       handler()
     } catch {
       case e: Exception =>
         logError(s"Error while handle${message.`type`()} $message", e)
-        client.getChannel.writeAndFlush(new RpcFailure(
-          requestId,
-          Throwables.getStackTraceAsString(e)))
-    } finally {
-      message.body().release()
```

Comment:
why are these lines deleted? This is needed to maintain the buffer count, right?

Reply:
`callback.onFailure(e)` covers  `client.getChannel.writeAndFlush(new RpcFailure(
          requestId,
          Throwables.getStackTraceAsString(e)))`
          
`message.body().release()` will be called in `TransportRequestHandler.processRpcMessage...`